packages/rollup-tests/src/failed-tests.json

```diff
@@ -707,29 +707,29 @@
   "rollup@function@wraps-object-expressions-as-statements: wraps object expressions that have become statements",
-  "rollup@sourcemaps@basic-support: basic sourcemap support@generates esm",
-  "rollup@sourcemaps@combined-sourcemap-with-loader: get combined sourcemap in transforming with loader@generates esm",
-  "rollup@sourcemaps@combined-sourcemap: get combined sourcemap in transforming@generates esm",
-  "rollup@sourcemaps@excludes-plugin-helpers: excludes plugin helpers from sources@generates esm",
-  "rollup@sourcemaps@existing-external-sourcemaps-combined: removes sourcemap comments@generates esm",
-  "rollup@sourcemaps@ignore-list-default: defaults to adding files within node_modules to the ignore list@generates esm",
-  "rollup@sourcemaps@ignore-list-false: accepts false for `sourcemapIgnoreList` to disable the default ignore-listing of node_modules@generates esm",
-  "rollup@sourcemaps@ignore-list-source-files: populates ignore list@generates esm",
-  "rollup@sourcemaps@ignore-list-sourcemap-path: correctly passes source map path@generates esm",
-  "rollup@sourcemaps@loaders: preserves sourcemap chains when transforming@generates esm",
-  "rollup@sourcemaps@names-transformed-render-chunk: names are recovered if transforms are used@generates esm",
-  "rollup@sourcemaps@names: names are recovered (https://github.com/rollup/rollup/issues/101)@generates esm",
-  "rollup@sourcemaps@reified-namespace: generates correct sourcemap with reified namespace (#668)@generates esm",
-  "rollup@sourcemaps@relative-paths: source paths are relative with relative dest (#344)@generates esm",
-  "rollup@sourcemaps@render-chunk-babili: generates valid sourcemap when source could not be determined@generates esm",
-  "rollup@sourcemaps@render-chunk: preserves sourcemap chains when transforming@generates esm",
-  "rollup@sourcemaps@single-length-segments: handles single-length sourcemap segments@generates esm",
-  "rollup@sourcemaps@sourcemap-base-url-without-trailing-slash: add a trailing slash automatically if it is missing@generates esm",
-  "rollup@sourcemaps@sourcemap-base-url: adds a sourcemap base url@generates esm",
-  "rollup@sourcemaps@sourcemap-file-hashed: populates file property of sourcemap with final name when using hashes@generates esm",
-  "rollup@sourcemaps@sourcemap-file: populates file property of sourcemap when plugins are used (#986)@generates esm",
-  "rollup@sourcemaps@transform-full-source-paths: provides the full source map path when transforming source maps@generates esm",
-  "rollup@sourcemaps@transform-low-resolution: handles combining low-resolution and high-resolution source-maps when transforming@generates esm",
-  "rollup@sourcemaps@transform-source-paths: transform sourcemap paths (#2168)@generates esm",
-  "rollup@sourcemaps@transforms: preserves sourcemap chains when transforming@generates esm"
+  "rollup@sourcemaps@basic-support: basic sourcemap support@generates es",
```

Comment:
I guess these titles don't need to be changed.

Reply:
It is generated by format type.